plugins/uniques.js

```diff
@@ -411,50 +588,71 @@ window.plugin.uniques.setupContent = function() {
+		+ '<label><input type="checkbox" id="droneVisited" onclick="window.plugin.uniques.updateDroneVisited($(this).prop(\'checked\'))"> Drone visited</label>'
 		+ '</div>';
 	plugin.uniques.disabledMessage = '<div id="uniques-container" class="help" title="Your browser does not support localStorage">Plugin Uniques disabled</div>';
 }
-
+// ***************************************************************************************
 window.plugin.uniques.setupPortalsList = function() {
 
 	window.addHook('pluginUniquesUpdateUniques', function(data) {
 		var info = plugin.uniques.uniques[data.guid];
-		if(!info) info = { visited: false, captured: false };
+		if (!info) info = { visited: false, captured: false, scoutControlled: false, droneVisited: false	};
 
 		$('[data-list-uniques="'+data.guid+'"].visited').prop('checked', !!info.visited);
 		$('[data-list-uniques="'+data.guid+'"].captured').prop('checked', !!info.captured);
+		$('[data-list-uniques="'+data.guid+'"].scoutControlled').prop('checked', !!info.scoutControlled);
+		$('[data-list-uniques="'+data.guid+'"].droneVisited').prop('checked', !!info.droneVisited);
 	});
 
 	window.addHook('pluginUniquesRefreshAll', function() {
 		$('[data-list-uniques]').each(function(i, element) {
 			var guid = element.getAttribute("data-list-uniques");
 
 			var info = plugin.uniques.uniques[guid];
-			if(!info) info = { visited: false, captured: false };
+			if (!info) info = { visited: false, captured: false, scoutControlled: false, droneVisited: false };
 
 			var e = $(element);
-			if(e.hasClass('visited')) e.prop('checked', !!info.visited);
-			if(e.hasClass('captured')) e.prop('checked', !!info.captured);
+			if (e.hasClass('visited')) e.prop('checked', !!info.visited);
+			if (e.hasClass('captured')) e.prop('checked', !!info.captured);
+			if (e.hasClass('scoutControlled')) e.prop('checked', !!info.scoutControlled);
+			if (e.hasClass('droneVisited')) e.prop('checked', !!info.droneVisited);
 		});
 	});
 
 	function uniqueValue(guid) {
 		var info = plugin.uniques.uniques[guid];
-		if(!info) return 0;
+		if (!info) return 0;
+		if (info.visited === undefined) return 0;
+		if (!info.visited) return 0;
+		if (info.visited && info.captured) return 2;
+		if (info.visited) return 1;
+	}
+
+	function scoutControlledValue(guid) {
+		var info = plugin.uniques.uniques[guid];
+		if (!info) return 0;
+		if (info.scoutControlled === undefined ) return 0;
+		if (info.scoutControlled === true) return 1;
+	}
 
-		if(info.visited && info.captured) return 2;
-		if(info.visited) return 1;
+	function droneVisitedValue(guid) {
+		var info = plugin.uniques.uniques[guid];
+		if (!info) return 0;
+		if (info.droneVisited === undefined ) return 0;
+		if (info.droneVisited === true) return 1;
 	}
 
 	window.plugin.portalslist.fields.push({
-		title: "Visit",
+		title: "V/C",
```

Comment:
Visit/capture/scout will already be in portal-list (but drone/scanned aren't)
Don't you want to replace/edit the columns instead of duplicate them ?

Reply:
> Don't you want to replace/edit the columns instead of duplicate them ?

Reuse, not replace.
E.g. use cached uniques data when there are no info from intel.
And highlight text in cases when data differ (with more explanation in tooltip).